### PR TITLE
Fixed svg issue for P_2_1_1_04

### PR DESCRIPTION
**Pull request by joeyklee**
Changes:
* added `width="100" height="100"` to svg tags
* updated preload of svg modules to ensure each svg is loaded 
* fixes: #123

### CODE REVIEW

**Comment by b-g** [comment]
Thanks! But did you really tackle all svg files and is 100 x 100 the correct size? 

screen shot 2017-09-04 at 08 42 55

**Comment by joeyklee** [comment]
Ah! I only did the svg files for that particular sketch. I will take care of the rest.

Regarding the size: 

+ now looking into the viewbox parameters (e.g. `viewBox(-205 207 100 100)`), I realized that the last 2 values specify the width and height of the svg file. I will use those to indicate the width and height of the svg.

**Comment by joeyklee** [comment]
Also fixed the loadImage() to array issue with 2e4563b45897cfc4b7870225711880b5a638d098

**Comment by joeyklee** [comment]
All the svgs should also now have width and height attributes:  see 3849719b8b7c13ace9a7638c2263f03f23e76ce0

**Comment by joeyklee** [comment]
ok - should be ready to go :)

**Comment by b-g** [comment]
@joeyklee Great + Many thanks! Please check for the "_01" sketches (whether we have to change anything in the book) ... then please merge.

**Comment by joeyklee** [comment]
@b-g I just did a check and it looks like the following are "_01" sketch code sections will need to be synchronized for the book:

* http://localhost:9966/01_P/P_2_3_4_01/
* http://localhost:9966/01_P/P_2_3_6_01/

should be a simple change to just move the array assignment `= []` to the `preload` for the sketches above.

screen shot 2017-09-05 at 20 01 10

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
 bower_components/
 node_modules/
+<<<<<<< HEAD
+data/
+.DS_Store
+=======
 /data/
+>>>>>>> generative-design/master
```
Comment (b-g) [inline .gitignore]: @joeyklee please resolve merge conflict

**Comment by b-g** [comment]
@joeyklee Many thanks! Just double check whether printed source code is affected ... it is not. Ready to merge after the merge conflict of `.gitignore` is resolved ... Great!!

**Comment by joeyklee** [comment]
@b-g - alrighty! should be solved with f7ec0bb1f95da04963083900bcfb647dc99ee4c5

**Pull request merged**